Parser/pegen.c

```diff
@@ -1259,6 +1259,7 @@ _PyPegen_check_tokenizer_errors(Parser *p) {
                 if (p->tok->level != 0) {
                     int error_lineno = p->tok->parenlinenostack[p->tok->level-1];
                     if (current_err_line > error_lineno) {
+                        PyErr_Clear();
```

Comment:
Is there any chance this code path might be hit with some other kind of error (e.g IndentationError, UnicodeDecodeError, MemoryError etc)? If not LGTM

Reply:
Nop, only `PyExc_SyntaxError`